### ISSUE TITLE
Make `pos` relative to pixel size

### ISSUE THREAD

**Issue by filipemeneses** [stagas/pixel-art#1]
Take the following code as an example:

```
    pixel.art([
      'aaaaaabcbbbbb',
      'bbaaabbcbbbbb',
      '  aaabcbbb   '
    ])
    .palette({
      'c': '#928457',
      'b': '#c1b388',
      'a': '#d6ceb6'
    })
    .pos({ x: 3, y: 17 })
    .scale(6)
    .draw(ctx)
```

The `{x, y}` position are following exact pixel size instead the scaled pixel size.

**Comment by filipemeneses** [comment]
I've made a pull request changing this.

**Comment by stagas** [comment]
Solved in bfeaf6db364c69bc95c088fed47d2333543dbf24

**Comment by filipemeneses** [comment]
Can you npm publish this update please?

**Comment by stagas** [comment]
@filipemeneses published as [v1.2.0](https://github.com/stagas/pixel-art/commit/0bca25297084659ceab6fabcf1f573bd5ced8ad3)

**Comment by filipemeneses** [comment]
Thank you
